sycl/include/sycl/accessor.hpp

```diff
@@ -2643,13 +2666,16 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS accessor<
 
 template <typename DataT, int Dimensions = 1>
 class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(local_accessor) local_accessor
-    : public local_accessor_base<DataT, Dimensions, access::mode::read_write,
+    : public local_accessor_base<DataT, Dimensions, detail::accessModeFromConstness<DataT>(),
                                  access::placeholder::false_t>,
       public detail::OwnerLessBase<local_accessor<DataT, Dimensions>> {
 
-  using local_acc =
-      local_accessor_base<DataT, Dimensions, access::mode::read_write,
-                          access::placeholder::false_t>;
+  using local_acc = local_accessor_base<DataT, Dimensions, detail::accessModeFromConstness<DataT>(),
+                                        access::placeholder::false_t>;
```

Comment:
The formatting here looks off. Don't we have >80 character lines in the above changes? Weird that lint doesn't pick that up.

Reply:
I always use git clang-format.  I'll fix it.

Reply:
thanks @steffenlarsen,   done